frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -255,9 +258,13 @@ export const dashboardLogic = kea<dashboardLogicType>([
                     actions.abortAnyRunningQuery()
 
                     try {
-                        return await api.update(`api/projects/${values.currentTeamId}/dashboards/${props.id}`, {
-                            filters: values.filters,
-                        })
+                        const dashboard: DashboardType<InsightModel> = await api.update(
```

Comment:
probably a dumb question, why do all API responses still return `InsightModel` and not `QueryBasedInsightModel`? Do we need to wait on that till we run the db migration script

Reply:
No it's a good question. Basically `QueryBasedInsightModel` is everything in `InsightModel`, except `filters`. This helped me see where we were still accessing filters in a converted model. 

The reason that the backend endpoint are still typed as `InsightModel`, is because they actually are. The backend side conversion is totally separate from the frontend side one. And you're right, we can change them once we ran the db migration script.